### PR TITLE
Bump flex to ^1.17, v1.13 - old infrastructure shutdown

### PR DESCRIPTION
**Pull request by em411**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | master          |
| Bug fix?        | yes                                                       |
| New feature?    | yes                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | yes |
| Related tickets | -                     |
| License         | MIT                                                          |

https://symfony.com/blog/the-old-flex-infrastructure-is-shutting-down

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -229,7 +229,7 @@
         "symfony/browser-kit": "^5.4",
         "symfony/debug-bundle": "^5.4",
         "symfony/dotenv": "^5.4",
-        "symfony/flex": "^1.7",
+        "symfony/flex": "^1.17",
```
Comment (stloyd) [inline composer.json:232]:
```suggestion
        "symfony/flex": "^1.17.1",
```
According to the blog post.

**Review by stloyd — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (Ferror) [reply composer.json:232]: Just to clarify :) https://symfony.com/blog/upgrade-flex-on-your-symfony-projects

Comment (stloyd) [reply composer.json:232]: Yes, exactly this part:
> But if the versions entry is showing a version that **is less than v1.17.1**, then, please keep reading, this blog post is for you.

**Comment by Zales0123** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "patch-2"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by Zales0123** [comment]
Hello @em411! I've allowed myself to change the PR to base on 1.10 branch and bump the flex constraint to 1.17.1 according to @stloyd comment. Thank you, for your contribution! 🎉

**Review by loic425 — APPROVED** [review]

**Comment by em411** [comment]
Hi @Zales0123, sorry for the delay on my side, I didn't saw the response :) 
Thanks for your help!

**Review by lchrusciel — APPROVED** [review]

**Review by Zales0123 — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you Emil! 🎉

**Pull request merged**